src/fuse_api.pxi

```diff
@@ -467,6 +467,22 @@ cdef session_loop_mt(workers):
         stdlib.free(wd)
 
 
+def stop():
+    '''Stop the FUSE main loop.
+
+    This function is thread safe. Note once the `main` function has returned
```

Comment:
Either "Note *that* once the `main`..." or just "Once the main function.."

Reply:
(re)done ^^